### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by joohaeng** [xinghaochen/Pose-REN#7]
I'm very interested in training Pose-REN from the scratch.

Do we need a special code for this, which is not included in this GitHub, or is it possible to train with a well-defined prototxt accompanied with well-augmented data?

If a control code is required, any outline hint will be appreciated!

Best,

Joo-Haeng

**Comment by xinghaochen** [comment]
Hi, you can easily train Pose-REN from scratch with a well-defined prototxt accompanied with well-augmented data. Here are some hints you may find helpful:
1. train the Init-CNN (baseline) with training data.
2. generate predicted labels of Init-CNN for training data and test set.
3. prepare the above labels from Init-CNN as guided data accompanied with original data.
4. train the Pose-REN ( use the weights of Init-CNN to initialize the network)

**Comment by joohaeng** [comment]
Great!

When training the main Pose-REN (not the baseline), do we also need to consider iterations, which is the key process in evaluation phase?

**Comment by xinghaochen** [comment]
Training other iterations of Pose-REN is similar with training the first iteration (guided with Init-CNN), except that we use predicted labels from the previous iteration as guided poses.

It should be noted that in order to use only one final model for Pose-REN in the inference stage, we use incremental training data to train Pose-REN in the second (and above) stage. You can find more detail in Section 3.4 in [our paper](https://arxiv.org/pdf/1708.03416.pdf).
